### PR TITLE
Update Hector documentation

### PR DESCRIPTION
**Pull request by kdorheim**
Here I have updated the Hector documentation to be consistent with the version of Hector used by GCAM. Much of this will have to be updated when upgrading to Hector v3.

### CODE REVIEW

File: hector.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
 layout: index
-title: Earth System Module – Hector v2.0	
+title: Earth System Module – Hector v2.5.0	
 prev: water.html
 next: emissions.html
 gcam-version: v6
 ---
 
-This section describes the carbon-cycle climate module - Hector - that is available for use in GCAM. MAGICC5.3 (Wigley, 2008) has traditionally been the only climate module available in GCAM. Hector v2.0 is the default climate model (Hartin et al., 2015) within GCAM.  Users still have the option of running MAGICC5.3 in GCAM5.1, however, we will not be supporting this option going forward.     
+This section describes the carbon-cycle climate module - Hector - that is available for use in GCAM. MAGICC5.3 (Wigley, 2008) has traditionally been the only climate module available in GCAM. Hector v2.5.0 is the default climate model (Hartin et al., 2015) within GCAM.  Users still have the option of running MAGICC5.3 in GCAM5.1, however, we will not be supporting this option going forward.
```
Comment (bpbond) [inline hector.md]: Perhaps

>Users still have the option of running MAGICC5.3 in GCAM5.1, but we will not be supporting this option going forward.

File: hector.md [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
 
-There most notable change between Hector v1.1 and Hector v2.0 is the inclusion of a one-dimensional ocean heat diffusion model - DOECLIM (Kriegler, 2005; Tanaka and Kriegler, 2007). With this addition, Hector v2.0 exhibits improved vertical ocean heat uptake, as well as surface response to radiative forcing. (https://github.com/JGCRI/hector/pull/206)
+There most notable change between Hector v1.1 and Hector v2.5.0 is the inclusion of a one-dimensional ocean heat diffusion model - DOECLIM (Kriegler, 2005; Tanaka and Kriegler, 2007). With this addition, Hector v2.5.0 exhibits improved vertical ocean heat uptake, as well as surface response to radiative forcing. (https://github.com/JGCRI/hector/pull/206)
```
Comment (bpbond) [inline hector.md:13]: What does "surface response" mean?

File: hector.md [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
 {: .fig}
 
 ## GCAM-Hector interactions
-Currently the GCAM sectors interact with Hector via their emissions.  At every time step, emissions from GCAM are passed to Hector. Hector converts these emissions to concentrations when necessary, and calculates the associated radiative forcing, as well as the response of the climate system (e.g., temperature, carbon-fluxes, etc.)  
+Currently the GCAM sectors interact with Hector via emissions.  At every time step, emissions from GCAM are passed to Hector. Hector converts these emissions to concentrations when necessary, and calculates the associated radiative forcing, as well as the response of the climate system (e.g., temperature, carbon-fluxes, etc.). Hector's climate information can be used as a climate constraint for in a [GCAM policy run](policies.html).
```
Comment (bpbond) [inline hector.md]: >of the climate and earth system (e.g., temperature, carbon-fluxes, etc.). Hector's climate information can be used as a climate constraint in a [GCAM policy run](policies.html).

File: hector.md [hunk 4]
```diff
@@ -89,27 +89,41 @@ For users who are running GCAM with the Mac or Windows Release Package, Hector s
...
+Hector is a flexible, simple climate model. Users can run Hector under various configurations, parameters, and constraints. 
+
+**Hector Configurations** 
+
+By default, Hector's carbon cycle model treats the entire land surface as a single, homogeneous ecosystem. However, it is possible to introduce some land surface heterogeneity by splitting the land surface into several different biomes with distinct parameters. This is explained in detail [here](https://jgcri.github.io/hector/articles/multiple-biomes.html).
+
+
+**Hector Parameters** 
+
+Hector has many parameters can be user adjusted, see the [online Hector manual](https://jgcri.github.io/hector/) for more information.
```
Comment (bpbond) [inline hector.md]: Semicolon, not comma, here.

**Review by bpbond — APPROVED** [review]
Looks good! A few minor suggestions to consider.

File: hector.md (hunk 2 above)
Comment (kdorheim) [reply hector.md:13]: Temperature change in response to radiative forcing?

**Pull request merged**